### PR TITLE
WIP: coh.ewi.tudelft.nl

### PR DESCRIPTION
**Pull request by JoepdeJong**
WIP for coh.ewi.tudelft.nl

- [ ] Find out how to migrate current wp-content

### CODE REVIEW

**Comment by JoepdeJong** [comment]
@julian9499 any ideas how we can migrate the current wp-content to kubernets?

**Comment by julian9499** [comment]
We would probably need a space claim like the study visit has now

**Comment by JoepdeJong** [comment]
> We would probably need a space claim like the study visit has now

Yeah I noticed, use the same config as studyvisit for COH.

What do you prefer? Migrating first everything from Rob to flux 1 and then to flux 2, or directly to flux2?

**Comment by julian9499** [comment]
I would do the migrating to flux1 for now. It's nice to know that everything will run on kubernetes well. Then we can look into migrating to a new v

File: coh/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                  key: password
+                  name: cie_coh-db-password
+            - name: WORDPRESS_DB_PREFIX
+              value: wp_
+            - name: APACHE_LOG_DIR
+              value: /var/log/apache/
+          securityContext:
+            runAsNonRoot: false
+          volumeMounts:
+            - name: coh-storage
```
Comment (JoepdeJong) [inline coh/deploy.yaml:77]: @julian9499 do we need more configuration for the spaceclaim than this?

And how to add the content? SSH into the pod and copy and paste?

Comment (julian9499) [reply coh/deploy.yaml:77]: I think so, but otherwise we can do the same as we did for the study visit which is make it with kubectl

**Pull request merged**